recipes/r-pspecterlib/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "pspecterlib" %}
+{% set version = "1.1.0" %}
+{% set github = "https://github.com/EMSL-Computing/pspecterlib" %}
+
+package:
+  name: "r-{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "{{ github }}/archive/refs/tags/{{ version }}.tar.gz"
+  sha256: 8ca6c456365f815bb1c80feecca5bbb8d32e785fab2a1a7be06574c1e29bbcd2
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  script_env:
+    - RGL_USE_NULL=TRUE # avoid warning at test-time
+  script: R CMD INSTALL --build . 
+  run_exports:
+    - {{ pin_subpackage("r-pspecterlib", max_pin="x.x") }}
+  
+requirements:
+  host:
+    - r-base >=4.2
+    - bioconductor-rhdf5
+    - bioconductor-mzr
+    - bioconductor-msnbase
+    - r-lsa
+    - r-seqinr
+    - r-dplyr
+    - r-reshape2
+    - r-gtools
+    - r-readxl
+    - r-data.table
+    - r-plotly
+    - r-ggplot2
+    - r-magrittr
+    - r-purrr
+    - r-tidyr
+    - r-htmltools
+    - r-rcolorbrewer
+    - r-isopat
+    - r-chnosz
+    - r-cairo
+    - r-rmarkdown
+    - r-knitr
+    - bioconductor-rawrr
+  run:
+    - r-base >=4.2
+    - bioconductor-rhdf5
+    - bioconductor-mzr
+    - bioconductor-msnbase
+    - r-lsa
+    - r-seqinr
+    - r-dplyr
+    - r-reshape2
+    - r-gtools
+    - r-readxl
+    - r-data.table
+    - r-plotly
+    - r-ggplot2
+    - r-magrittr
+    - r-purrr
+    - r-tidyr
+    - r-htmltools
+    - r-rcolorbrewer
+    - r-isopat
+    - r-chnosz
+    - r-cairo
+    - r-rmarkdown
+    - r-knitr
+    - bioconductor-rawrr
+
+test:
+  commands:
+    - $R -e "library('pspecterlib')"
+
+about:
+  home: "{{ github }}"
+  license_file: license.txt
```

Comment:
You also need to include this file in the recipe (i.e. `cp ~/path/to/pspecterlib/licence.txt ~/path/to/bioconda/recipes/r-pspecterlib/`)

Reply:
I'm not sure I understand what you mean with the path - related to the bioconda root repository and then include the file in the submission or?

Reply:
so the license.txt must be copied from the source of pspecterlib to the recipe directory of bioconda (i.e. next to meta.yml here)

Reply:
@kdm9 why is this the case? This is at least not true in the general case. Conda-build is smart enough to find the license file in the source tarball and copy it automatically over.

Reply:
my apologies, this is what I'd always remembered been asked in recipes and had the linter bounce me for -- seems I'm wrong here so ignore me @hechth  :)